binnacle/lib/WindRequest.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:geolocator/geolocator.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'dart:async';
+
+class WindRequest {
+  final Wind wind;
+
+  WindRequest({this.wind});
+
+  factory WindRequest.fromJson(Map<String, dynamic> json) {
+    print("WindJson: "+json['wind'].toString());
+    return WindRequest(
+      wind: Wind.fromJson(json['wind'])
+    );
+  }
+}
+
+class Wind{
+  String speed;
```

Comment:
Should speed and heading be Strings? I feel like there is something more precise for this.